docs/howto/outside_controller_use.md

```diff
@@ -0,0 +1,43 @@
+## Using AMS Outside Of A Controller
+
+### Serializing a resource 
+
+In the current version of AMS, serializing resources outside controllers is
+fairly simple:
+
+```ruby
+# Create our resource
+post = Post.create(title: "Sample post", body: "I love Active Model Serializers!")
+
+# Optional options parameters
+options = {}
```

Comment:
TODO: link to description of `options` and how they are used ( as serializer_options, adapter_options, and renderer options)